### PR TITLE
[bluray] Improve Blu-ray playback

### PR DESCRIPTION
**Pull request by ace20022**

### CODE REVIEW

File: xbmc/cores/VideoPlayer/IVideoPlayer.h [hunk 1]
```diff
@@ -52,7 +51,7 @@ struct SStartMsg
 class IVideoPlayer
 {
 public:
-  virtual int OnDVDNavResult(void* pData, int iMessage) = 0;
+  virtual int OnDiscNavResult(void* pData, int iMessage) = 0;
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/IVideoPlayer.h]: Good Idea!

**Comment by ace20022** [comment]
jenkins build this please

**Comment by Hedda** [comment]
Will you also upgrade to the recently released libbluray 1.0.0 version?

http://git.videolan.org/?p=libbluray.git;a=blob;f=ChangeLog

That adds Java VM (BD-J) on MacOS + JVM paths for popular Linux distros.

ftp://ftp.videolan.org/pub/videolan/libbluray/1.0.0/

Congrats nonetheless on adding initial BD-J menu support to Kodi!

File: xbmc/system.h [hunk 2]
```diff
@@ -94,6 +94,7 @@
 #define HAS_WIN32_NETWORK
 #define HAS_IRSERVERSUITE
 #define HAS_FILESYSTEM_SMB
+#define HAVE_LIBBLURAY_BDJ
```
Comment (Rechi) [inline xbmc/system.h]: bluray is only an optional dependency so this definition should set by cmake if bluray is found

**Comment by ace20022** [comment]
jenkins build this please

**Comment by Hedda** [comment]
@ace20022  Will you also upgrade to the recently released libbluray 1.0.0 version?

http://git.videolan.org/?p=libbluray.git;a=blob;f=ChangeLog

That adds Java VM (BD-J) on MacOS + JVM paths for popular Linux distros.

ftp://ftp.videolan.org/pub/videolan/libbluray/1.0.0/

Congrats nonetheless on adding initial BD-J menu support to Kodi!

**Comment by popcornmix** [comment]
@ace20022 
is it deliberate that the "Play main title" option is removed by this PR?
See: http://forum.kodi.tv/showthread.php?tid=298461&pid=2553863#pid2553863

**Comment by ace20022** [comment]
@popcornmix No it's not. Thx for the ping.

**Pull request merged**